contractcourt/chain_arbitrator.go

```diff
@@ -232,6 +236,9 @@ func newActiveChannelArbitrator(channel *channeldb.OpenChannel,
 		ShortChanID: channel.ShortChanID(),
 		BlockEpochs: blockEpoch,
 		ForceCloseChan: func() (*lnwallet.LocalForceCloseSummary, error) {
+			if c.cfg.SafeMode {
```

Comment:
this will beed to be checked when the force close request is first sent. Otherwise I think we'll get into a state where we will attempt to force close the channel on every restart.

Reply:
indeed, and we should mark it before we mark the channel borked and save the force close, o/w we will broadcast if the node is restarted and we're not in safe mode

Reply:
I don't think we should mark the channel at all in safe mode though, just to not alter any state?

Reply:
sorry, check* it before we mark

yes we want to make sure no state is altered